network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -280,6 +280,25 @@ def generate_card_grid_field():
     return generate_field("card_grid", {"cards": cards})
 
 
+def generate_mozfest_card_grid_field():
+    heading = fake.sentence(nb_words=4, variable_nb_words=True)
+
+    cards = []
+
+    for n in range(2):
+        cards.append(
+            {
+                "image": choice(Image.objects.all()).id,
+                "title": fake.paragraph(nb_sentences=1, variable_nb_sentences=False),
+                "body": fake.paragraph(nb_sentences=10, variable_nb_sentences=True),
+                "link_label": " ".join(fake.words(nb=3)),
+                "link_url": fake.url(schemes=["https"]),
+            }
+        )
+
+    return generate_field("mozfest_card_grid", {"cards": cards, "heading": heading})
+
+
```

Comment:
I was planning to not support this but I think it's going to help speed up Chris with the FE so I configured it, I think I will be able to do the other blocks too.